pkgs/by-name/go/goat-cli/package.nix

```
@@ -0,0 +1,35 @@
+{ lib,
+fetchFromGitHub,
+buildGoModule,
+fetchpatch
+}:
+
+buildGoModule rec {
+  pname = "goat-cli";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    repo = "goat";
+    owner = "studio-b12";
+    rev = "v${version}";
+    hash = "sha256-0rOGZEMSPIU3gK31X8uozhMh/82HGbcCpGOGKZ0qvrc=";
+  };
+
+  vendorHash = "sha256-DtEXgGYSkWO876so6LEOkhVwDt/zrflDZdY4O2lz1mw=";
+
+  patches = [
+    ./mock-fix.patch
+  ];
+
+  # Checks currently fail because of an issue with github.com/studio-b12/goat/mocks
+  doCheck = false;
+
+  meta = with lib; {
+    description = "An integration testing tool for HTTP APIs uisng a simple script language.";
```

The dot at the end should be removed (:
```suggestion
    description = "An integration testing tool for HTTP APIs uisng a simple script language";
```

---

fixed in https://github.com/NixOS/nixpkgs/commit/c90aeb96b1da9065cc36833e77f865be9bb804e0